dds_web/database/models.py

```diff
@@ -73,38 +20,25 @@ class Facility(db.Model):
     public_id = db.Column(db.String(50), unique=True, nullable=False)
     name = db.Column(db.String(100), unique=True, nullable=False)
     internal_ref = db.Column(db.String(10), unique=True, nullable=False)
-    username = db.Column(db.String(20), unique=True, nullable=False)
-    password = db.Column(db.String(120), unique=False, nullable=False)
-    # settings = db.Column(db.String(50), unique=False, nullable=False)
-    # email = db.Column(db.String(80), unique=True, nullable=False)
-    # phone = db.Column(db.String(20), unique=False, nullable=True)
     safespring = db.Column(db.String(120), unique=False, nullable=False)  # unique=True later
 
     # Relationships
-    user_projects = db.relationship(
-        "Project",
-        backref="project_facility",
-        lazy=True,
-        foreign_keys="Project.facility",
-    )
+    # One facility can have many users
+    users = db.relationship("User", backref="facility")
+    # One facility can have many projects
+    projects = db.relationship("Project", backref="responsible_facility")
 
     def __repr__(self):
         """Called by print, creates representation of object"""
 
-        return f"<Facility {self.username}>"
+        return f"<Facility {self.public_id}>"
 
 
-class Role(db.Model):
-    """Data model for roles - used to find correct table."""
-
-    # Table setup
-    __tablename__ = "roles"
-    __table_args__ = {"extend_existing": True}
-
-    # Columns
-    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
-    username = db.Column(db.String(20), unique=True, nullable=False)
-    facility = db.Column(db.Boolean, unique=False, nullable=False)
+project_users = db.Table(
```

Comment:
Would it be better to name it `user_projects` ?

Reply:
I mean, we could technically, but since it's a many-to-many table, and the `User` table has a `relationship` and not a column with foreign key, it's more logical I feel to have it as `project_users`. The original idea was to have an association table to keep track of which users are allowed to access which projects. `user_projects` is basically `projects` relationship in the `User` table already.

Reply:
Its fine, I was just talking in terms of semantics 👍

Reply:
Me too:

- `User.projects` : The projects a specific user has access to
- `project_users`: Which users have access to specific projects